cmd/cstor-volume-mgmt/volume/volume.go

```diff
@@ -107,11 +107,11 @@ func extractReplicaStatusFromJSON(str string) (*apis.CVStatus, error) {
 func CreateIstgtConf(cStorVolume *apis.CStorVolume) []byte {
 
 	var buffer bytes.Buffer
-	buffer.WriteString(`# Global section
+	_, _ = buffer.WriteString(`# Global section
```

Comment:
Shall we get rid of all `_,_` ? OR alternatively, we can add variables to retain the values.

Reply:
Thanks for review. I will update with variables